vllm/utils.py

```diff
@@ -331,22 +330,6 @@ def get_cpu_memory() -> int:
     return psutil.virtual_memory().total
 
 
-def seed_everything(seed: int) -> None:
-    """
-    Set the seed of each random module.
-
-    Loosely based on: https://github.com/Lightning-AI/pytorch-lightning/blob/2.4.0/src/lightning/fabric/utilities/seed.py#L20
-    """
-    random.seed(seed)
-    np.random.seed(seed)
-
-    if current_platform.is_cuda_alike():
-        torch.cuda.manual_seed_all(seed)
-
-    if current_platform.is_xpu():
-        torch.xpu.manual_seed_all(seed)
```

Comment:
@wangshuai09 it seems this part of logic is missing

Reply:
I have found this logic is implement in `torch.manual_seed`, https://pytorch.org/docs/stable/_modules/torch/random.html#manual_seed. The pytorch_lightning also remove `manual_seed_all` in this PR: https://github.com/Lightning-AI/pytorch-lightning/commit/dd206a20d8cbe037912c144fc3c5e5306ac0d77e.